Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -128,7 +128,7 @@ public NodeSettings()
 
             // The logger factory goes in the settings with minimal configuration, 
             // that's so the settings can also log out its progress.
-            nodeSettings.LoggerFactory.AddConsoleWithFilters();
+            nodeSettings.LoggerFactory.AddConsoleWithFilters(out var consoleSettings);
```

Comment:
wow i did not know this is possible and would compile, this is sick :-)
not sure yet if i like it, but thanks for new thingy